bosh_cli/lib/cli/commands/ssh.rb

```diff
@@ -6,7 +6,7 @@ module Command
     class Ssh < Base
 
       # bosh ssh
-      usage 'ssh'
+      usage 'ssh <job>/<index>'
```

Comment:
My guess at what you are trying to achieve here is to have `bosh help scp` print out the parameters that can be passed into that command. `usage` is probably not meant to be used this way. Other commands, such as `bosh deployment` or `bosh login` print their parameters, the printing even considers if the parameters are optional or not. To achieve this, they use some funky logic hidden in https://github.com/cloudfoundry/bosh/blob/master/bosh_cli/lib/cli/commands/help.rb#L62-L74

The parsing of method parameters happens in https://github.com/cloudfoundry/bosh/blob/master/bosh_cli/lib/cli/command_handler.rb#L66-L74.

So the problem seems to be that `bosh ssh` and `bosh scp` accept `*args` instead of specifying what might go in there – if these methods were taking specific arguments, you'd get the documentation for free.

Note that there is some funky comparisons going on with `cmd.usage` in https://github.com/cloudfoundry/bosh/blob/master/bosh_cli/lib/cli/commands/help.rb#L40 and https://github.com/cloudfoundry/bosh/blob/master/bosh_cli/lib/cli/commands/help.rb#L51 so your change might add some unexpected sideeffects.

The configuration adds commands with `usage` as key: https://github.com/cloudfoundry/bosh/blob/master/bosh_cli/lib/cli/config.rb#L39 and this is used in parsing the commandline to find out which command to run: https://github.com/cloudfoundry/bosh/blob/master/bosh_cli/lib/cli/runner.rb#L214-L218

So I'd rather not have the `usage` contain parameters and rather have some real arguments in the method itself.